pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -126,6 +147,10 @@ public static double getTotalCpuLimit(boolean isCGroupsEnabled) {
                 if (quota > 0) {
                     return 100.0 * quota / period;
                 }
+                int totalCpuCount = getTotalCpuCount();
```

Comment:
Actually, java 17 tracks container-aware resource usage. I think we just need to read jvm cpu limit and load if the jvm already provides the container-aware cpu limit and load.

Reference:
https://developers.redhat.com/articles/2022/04/19/java-17-whats-new-openjdks-container-awareness